manifests/t/ThePyzoteam/pyzo/4.2.1/ThePyzoteam.pyzo.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ThePyzoteam.pyzo
+PackageVersion: 4.2.1
+PackageLocale: en-US
+Publisher: The Pyzo team
+PublisherUrl: https://github.com/pyzo
+PublisherSupportUrl: https://github.com/pyzo/pyzo/issues
+Author: pyzo
+PackageName: pyzo
+PackageUrl: https://github.com/pyzo/pyzo
+License: priority

```

```suggestion
License: BSD-2-Clause
```
License Text matches https://spdx.org/licenses/BSD-2-Clause.html